config_defaults.yml

```diff
@@ -9,7 +9,14 @@ Gemfile:
   - gem: rspec-puppet-facts
     version: '>= 1.7'
   - gem: puppetlabs_spec_helper
-    version: '>= 0.8.0'
+    version: ">= 1.1.0', '<= 2.1.0"
+  - gem: parallel_tests
+    options:
+      platforms:
```

Comment:
For the 1.15 cycle we will continue support puppet 3 and on some platforms that runs on ruby 1.9. We will branch and pin there, so master will soon move to puppet 4 and drop it.